target/linux/ipq806x/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -49,7 +49,7 @@ case "$FIRMWARE" in
 	case $board in
 	askey,rt4230w-rev6)
 		caldata_extract "art" 0x5000 0x7f1f
-		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary ART 0x6) +2)
+		ath10k_patch_mac $(macaddr_add $(mtd_get_mac_binary ART 0x5006))
```

Comment:
See above: If you read from 0x5006 here, you should be able to remove the second line entirely.

Reply:
Yeah it worked. I was just going off what all the other devices did